tests/foreman/cli/test_errata.py

```diff
@@ -163,18 +166,13 @@ def setUp(self):
             'host-collection-id': self.host_collection['id'],
             'organization-id': self.org['id']
         })
-        if bz_bug_is_open('1405434'):
-            # install the custom package for each host
-            for virtual_machine in self.virtual_machines:
-                virtual_machine.run(
-                    'yum install -y {0}'.format(self.CUSTOM_PACKAGE)
-                )
-        else:
-            HostCollection.package_install({
-                'id': self.host_collection['id'],
-                'organization-id': self.org['id'],
-                'packages': [self.CUSTOM_PACKAGE]
-            })
+        for virtual_machine in self.virtual_machines:
+            virtual_machine.run(
+                'yum install -y {0}'.format(self.CUSTOM_PACKAGE)
+            )
+            result = virtual_machine.run(
+                'rpm -q {0}'.format(self.CUSTOM_PACKAGE))
+            self.assertEqual(result.return_code, 0)
```

Comment:
Any comments about these changes? You've done basically opposite - you're proposing to apply workaround, instead of installing package to entire host collection as bz was resolved.